tests/Microsoft.DotNet.Framework.Docker.Tests/projects/webapp-3.5/Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG BASE_BUILD_IMAGE
+
+FROM $BASE_BUILD_IMAGE
+
+WORKDIR /app
+COPY . ./
+
+RUN msbuild SimpleWebApplication.csproj /verbosity:quiet /p:Configuration=Release
+
+ENTRYPOINT dir ./bin/SimpleWebApplication.dll
```

Comment:
The ENTRYPOINT doesn't seem like a real usage scenario.  If you want to capture this, it seems like it would be better done with a `docker run`.

Reply:
I think the more canonical way to do this would be do use `docker run` after you built your Dockerfile - `docker run dir ./bin/SimpleWebApplication.dll`

Reply:
>I am trying to make sure that that build was successful.

How is this being verified?  I'm not seeing any automation that verifies the build was successful other than the Dockerfile built without error.  Perhaps that is adequate, I am trying to understand.

Reply:
> I don't see how "docker run" is different from using CMD or ENTRYPOINT. I reverted to my original approach, which is using dir, which verifies right after msbuild.

The responsibility of the Dockerfile in a scenario like this is to build the application.  Setting the ENTRYPOINT to list the contents on a directory doesn't make sense and makes the resulting image more difficult to use for anything else as you would need to override the ENTRYPOINT. 

Where is the new PR?  I am not finding it?  Why not just update this PR?